src/registry.ts

```diff
@@ -143,5 +159,13 @@ export async function setEntry(
     ...opts,
     method: "post",
     data,
+    // Transform the request to remove quotes, since the revision needs to be
+    // parsed as a uint64 on the Go side.
+    transformRequest: function (data: unknown) {
```

Comment:
To be safe we probably need to do some type checking on `data` here. In the current implementation I guess it's not necessary, because we're applying it only to a subset of the requests, but if we were to use the `json-bigint` library it would become generic and get called on all requests and responses and we'd have to verify `data` is an object here and then call `JSONbig.stringify(data)` on it.

I only glanced at the documentation though, but there's bound to be quite some articles on how to do it. Here are a couple I found:
https://medium.com/@itstalwar15/handling-bigint-in-axios-using-json-big-in-javascript-d915ae85ffc0
https://stackoverflow.com/questions/43787712/axios-how-to-deal-with-big-integers
https://github.com/axios/axios/issues/983

Reply:
That looks pretty elegant. If we ever have more uint64s in the API we can switch over. For now I'd prefer not to introduce another dependency.

In the present code I don't think type checking is required here